src/react/hooks/paragon/useParagonThemeVariants.js

```diff
@@ -0,0 +1,196 @@
+            }
+            const otherExistingLinks = getExistingThemeVariantLinks(isBrandOverride);
+            removeExistingLinks(otherExistingLinks);
+            return;
+          }
+          const paragonThemeAccessor = isBrandOverride ? 'brand' : 'paragon';
+          const themeUrls = PARAGON_THEME?.[paragonThemeAccessor]?.themeUrls ?? {};
+          if (themeUrls.variants && themeUrls.variants[themeVariant]) {
+            const themeVariantFallbackUrl = `${getConfig().BASE_URL}/${themeUrls.variants[themeVariant].fileName}`;
+            logInfo(`Falling back to locally installed theme variant (${themeVariant}) CSS: ${themeVariantFallbackUrl}`);
+            themeVariantLink = createThemeVariantLink(themeVariantFallbackUrl, {
+              isFallbackThemeUrl: true,
+              isBrandOverride,
+            });
+            const otherExistingLinks = getExistingThemeVariantLinks(isBrandOverride);
+            removeExistingLinks(otherExistingLinks);
+            const foundParagonThemeVariantLink = getParagonThemeVariantLink();
+            if (foundParagonThemeVariantLink) {
+              foundParagonThemeVariantLink.insertAdjacentElement(
+                'afterend',
+                themeVariantLink,
+              );
+            } else {
+              document.head.insertAdjacentElement(
+                'afterbegin',
+                themeVariantLink,
+              );
+            }
+          } else {
+            logError(`Failed to load theme variant (${themeVariant}) CSS from ${url} and locally installed fallback URL is not available. Aborting.`);
+            if (isBrandOverride) {
+              setIsBrandThemeVariantLoaded(true);
+            } else {
+              setIsParagonThemeVariantLoaded(true);
+            }
+          }
+        };
+        return themeVariantLink;
+      };
+
+      if (!existingThemeVariantLink) {
+        const paragonThemeVariantLink = createThemeVariantLink(value.urls.default);
+        document.head.insertAdjacentElement(
+          'afterbegin',
+          paragonThemeVariantLink,
+        );
+
+        if (value.urls.brandOverride) {
+          const brandThemeVariantLink = createThemeVariantLink(value.urls.brandOverride, { isBrandOverride: true });
+          const foundParagonThemeVariantLink = getParagonThemeVariantLink();
+          if (foundParagonThemeVariantLink) {
+            foundParagonThemeVariantLink.insertAdjacentElement(
+              'afterend',
+              brandThemeVariantLink,
+            );
+          } else {
+            document.head.insertAdjacentElement(
+              'afterbegin',
+              brandThemeVariantLink,
+            );
+          }
+        } else {
+          setIsBrandThemeVariantLoaded(true);
+        }
+      } else {
+        const updatedStylesheetRel = generateStylesheetRelAttr(themeVariant);
+        existingThemeVariantLink.rel = updatedStylesheetRel;
+        existingThemeVariantBrandLink.rel = updatedStylesheetRel;
```

Comment:
Need an extra conditional when brand link is not defined
```suggestion
        if (existingThemeVariantBrandLink) { existingThemeVariantBrandLink.rel = updatedStylesheetRel; }
```